addon/serializers/github-release.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import GithubSerializer from './github';
+
+export default GithubSerializer.extend({
+  keyForAttribute(attr) {
+    return Ember.String.decamelize(attr);
```

Comment:
👍 

As an aside, it seems like we could push this `keyForAttribute` override into the base serializer and delete a lot of code that does this manually